__tests__/integration/utils/test-utils-integration.js

```diff
@@ -794,18 +794,42 @@ export class TestUtils {
   static async waitNewBlock(currentHeight = null) {
     let baseHeight = currentHeight;
 
+    // This timeout is a protection, so the integration tests
+    // don't keep running in case of a problem
+    // After using the timeout as 120s, we had some timeouts
+    // because the CI runs in a free github runner
+    // so we decided to increase this timeout to 600s, so
+    // we don't have this error anymore
+    const timeout = 600000;
+    let timeoutReached = false;
+    // Timeout handler
```

Comment:
Improve or remove this comment

Reply:
Done https://github.com/HathorNetwork/hathor-wallet-headless/pull/354/commits/e7c3c00d7a86bf63295451174dc058e38a30b2c2